components/list/list-item-generic-layout.js

```diff
@@ -276,14 +288,18 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 	}
 
 	_focusNextWithinCell(node, num = 1) {
+
 		if (!node || (node.assignedSlot && node.assignedSlot === this._getThisCell())) return null;
 		let focusable = null;
 		let siblingNum = 1;
 		while (!focusable || siblingNum < num) {
 			node = this._getNextSiblingInCell(node);
+
 			if (!node) break;
 			++siblingNum;
+
 			focusable = isFocusable(node, true) ? node : getFirstFocusableDescendant(node);
+			if (isInteractiveDescendant(focusable)) focusable = null;
```

Comment:
We need to prevent the generic layout's logic from trying to focus on focusable elements within an interactive widget such as tag-list.  Without this check, the logic would find and focus on them because such widgets in non-interactive mode are not actually inert (because they are still interactive with the mouse), even though it should skip past them.